### DISCUSSION TITLE
NavigationView does not work as expected in Android as it in Uno-UWP

### DISCUSSION THREAD

**Discussion by rakeshpatel2318** [unoplatform/uno#8173] [General]
Current Behavior:
--------------------
I am currently developing small prototype using UNO-Platform and using NavigationView (for Hamburger kind of menu functinality) in UNO-UWP and Android Heads.

The NavigationView works as expected for UNO-UWP but NOT in UNO-Android.
The NavigationView menu overlaps (transparently) on all the controls in Mainpage even while the Navigation mode is set for Compact (i.e. only button visible but not Menu drawer)

When I set the the NavigationView on the main page grid row, I am NOT able to select any of the Mainpage controls.
If I don't set the NavigationView on the main page grid row then the navigation view control is rendered or displayed behind the dropdown and other controls in UNO-Android.

I have attached a sample code for further reference and analysis.

Expected Behavior:
---------------------
The NavigationView controls should work identically as it in UNO-UWP head.

[For UNO Team.zip](https://github.com/unoplatform/uno/files/8121431/For.UNO.Team.zip)

**Comment by francoistanguay** [comment]
We have a few examples of using the NavView in real apps including https://github.com/unoplatform/uno.extensions/tree/main/samples/Commerce and https://github.com/unoplatform/Uno.Gallery that might be useful?

**Comment by sasakrsmanovic** [comment]
@rakeshpatel2318  In addition, you may want to take a look at another sample implementation https://github.com/unoplatform/Uno.Ch9
